### DISCUSSION TITLE
How to create Ubuntu rootfs images

### DISCUSSION THREAD

**Discussion by chwzr** [firecracker-microvm/firecracker#3516] [Q&A] [answered]
Is there a guide wich shows how you create the rootfs image from the latest getting started docs?

https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.ext4

**Comment by chwzr** [comment, marked as answer]
Never mind, found the devtool command: 
```
./devtool build_rootfs -s 500M
```

**Comment by xmarcalx** [comment]
Closing this discussion as resolved.
